src/data/reducers/courseInfo.test.js

```diff
@@ -53,11 +53,13 @@ describe('courseInfo reducer', () => {
       });
   });
 
+  /* isFetching is `true` for all of the course create tests because course create shouldn't
+  alter isFetching. Since oldState has it set to true, course create will maintain the value */
```

Comment:
We should review this, It should not be true here.  I am also looking at this in my PR now.

Reply:
https://github.com/edx/publisher-frontend/pull/58/